utils/src/test/scala/cromwell/pipeline/utils/AkkaTestSources.scala

```diff
@@ -0,0 +1,22 @@
+package cromwell.pipeline.utils
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import org.scalatest.{ BeforeAndAfterAll, Suite }
+
+trait AkkaTestSources extends BeforeAndAfterAll {
+  this: Suite =>
+  lazy val actorSystem = ActorSystem("test")
```

Comment:
I think an `ActorSystem` can be marked as an `implicit` too.
In this case, you won't have to do weird things like
`implicit val system = actorSystem`
as you did somewhere above.

Reply:
did it